warehouse/models/mart/gtfs_quality/fct_daily_schedule_feed_validation_notices.sql

```diff
@@ -37,6 +41,14 @@ first_outcome_per_version AS (
     ) = 1
 ),
 
+-- each validation outcome is valid until the next version, or the end of the feed's validity
+outcomes_with_end_dt AS (
+    SELECT
+        *,
+        COALESCE(LEAD(DATE_SUB(extract_dt, INTERVAL 1 DAY)) OVER (PARTITION BY feed_key ORDER BY extract_dt), feed_valid_to_dt) AS outcome_valid_to
```

Comment:
In general I do not necessarily love hard-coding `Interval 1 day` kind of things. Why not `{{ make_end_of_valid_range(LEAD(extract_dt) ... ) }}`?

Reply:
Ah just because I was dealing with dates and not timestamps; I can make a duplicate macro for dates.